packages/rollup-plugin-import-meta-assets/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @web/rollup-plugin-import-meta-assets
 
+## 2.0.3
+
+### Patch Changes
+
+- Enabled using dynamic import vars when importing assets
```

Comment:
did you add it manually?
if yes, please remove it and also the version bump in `package.json`
and instead use `npm changelog` to make a patch changelog file

Reply:
`npx changeset` indeed 👍🏻

Reply:
@bashmish I'm a little new to changesets and changelogs. Do you want me to do `npx changeset` and commit or `npx changelog`? And does this count as a patch or a minor?

Reply:
yes, and then you see how the system will bump the version in the GitHub comment in your PR
the changelog will also include the text which will end up in CHANGELOG.md of the corresponding package

![image](https://github.com/modernweb-dev/web/assets/137844/3484cca4-b1e1-4290-b51a-36443cd427c2)